interceptors/logging/interceptors.go

```diff
@@ -50,64 +52,117 @@ func (c *reporter) PostCall(err error, duration time.Duration) {
 	logger.With(c.opts.durationFieldFunc(duration)...).Log(c.opts.levelFunc(code), fmt.Sprintf("finished %s %s call", c.kind, c.typ))
 }
 
-func (c *reporter) PostMsgSend(interface{}, error, time.Duration) {}
+// PostMsgSend logs the details of the servingObject that is flowing out of the rpc.
+// resp object wrt server.
+// Log the details of the request, skip if the object is response.
```

Comment:
```suggestion
// Log the details of the first request, skip if the object is response.
``` 
? Same on `PostMsgRecv`